README.md

```diff
@@ -57,55 +59,94 @@ This image only contains ModSecurity built from the code provided on the [ModSec
 
 ### TLS/HTTPS
 
-If you want to run your web traffic over TLS you can simply set the SETTLS argument to true when using Docker build. Note: This will use self signed certificates, to use your own certificates (recommended) COPY or mount (-v) your server.crt and server.key into /usr/local/apache2/conf/. Please remember you'll need to forward the HTTPS port
+The TLS is configured by default on port 443. Note: The default configuration uses self signed certificates, to use your own certificates (recommended) COPY or mount (-v) your server.crt and server.key into `/usr/local/apache2/conf/`. Please remember you'll need to forward the HTTPS port.
 
 ```
-$docker build  --build-arg SETTLS=True -t my-modsec .
-$docker run -p 8443:443 my-modsec
+$ docker build -t my-modsec .
+$ docker run -p 8443:443 my-modsec
 ```
 
 ### Proxying
 
-ModSecurity is often used as a reverse proxy. This allows one to use ModSecurity without modifying the webserver hosting the underlying application (and also protect web servers that modsecurity cannot currently embedd into). To set proxying one must set SETPROXY to True and provide the location of the Proxy via the PROXYLOCATION argument. Note: if you are going to be proxying to a HTTPS site, you'll need to set SETTLS to true also.
+ModSecurity is often used as a reverse proxy. This allows one to use ModSecurity without modifying the webserver hosting the underlying application (and also protect web servers that modsecurity cannot currently embedd into). The proxy is set by default to true and the location is defined by BACKEND environment variable. The SSL is enabled by default.
```

Comment:
Is there a variable to handle private TLS CAs? Otherwise maybe makes sense to clarify that certificates should be recognized by the installed CAs. (e.g. usage of SSLProxyVerify in apache)